pkg/pgmodel/migrate.go

```diff
@@ -111,18 +111,34 @@ func Migrate(db *pgxpool.Pool, versionInfo VersionInfo) (err error) {
 	return nil
 }
 
-//CheckDependcies makes sure all project dependencies are set up correctly
+// CheckDependcies makes sure all project dependencies, including the DB schema
+// the extension, are set up correctly. This will set the ExtensionIsInstalled
+// flag and thus should only be called once, at initialization.
 func CheckDependencies(db *pgxpool.Pool, versionInfo VersionInfo) (err error) {
+	conn, err := db.Acquire(context.Background())
+	if err != nil {
+		return fmt.Errorf("failed to check schema version: %w", err)
+	}
+	defer conn.Release()
+	err = CheckSchemaVersion(context.Background(), conn.Conn(), versionInfo)
+	if err != nil {
```

Comment:
Can be on the same line. Simple error returns on new lines add unnecessarily to the code length.

Reply:
__This__ one actually should be on the same line